package.json

```diff
@@ -41,23 +44,30 @@
   "dependencies": {
     "@ant-design/icons": "^4.6.2",
     "@babel/runtime": "^7.13.10",
+    "@craco/craco": "^6.1.2",
```

Comment:
I missed this before. Are we using craco? 

can you review that only required dependencies are added?